models/palm_detection_mediapipe/README.md

```diff
@@ -1,10 +1,11 @@
 # Palm detector from MediaPipe Handpose
 
-This model detects palm bounding boxes and palm landmarks, and is converted from Tensorflow-JS to ONNX using following tools:
+This model detects palm bounding boxes and palm landmarks, and is converted from TFLite to ONNX using following tools:
```

Comment:
By the way, TFLite support in DNN has been merged. Dev branch can now load and run TFLite models.